ui/src/styles.scss

```diff
@@ -17,3 +19,24 @@ body {
 .cursor-pointer {
   cursor: pointer;
 }
+.mat-button.mat-primary{
+  color: #3367D6;
+}
+.global-font-color{
+  color: #000000;
+}
+.body-font{
+font-size: 14px;
+}
+.primay-header{
+font-size: 20px;
+}
+.mat-stroked-button.mat-primary {
+  color: #3367D6;
+
+}
+.mat-raised-button {
```

Comment:
This class combination should be changed to **.mat-raised-button.mat-primary**. 
And all mat-raised-button in HTML should have **color="primary"**

Reply:
Made the suggested changes